src/include/stir/KOSMAPOSL/KOSMAPOSLReconstruction.h

```diff
@@ -173,11 +173,11 @@ class KOSMAPOSLReconstruction:
    one place, all objects that use the shared pointer will be affected.
   */
  //@{
-  void set_anatomical_prior_sptr(shared_ptr<TargetT>, int index);
+  void set_anatomical_prior_sptrs(shared_ptr<TargetT>, int index);
```

Comment:
sorry. this doesn't make sense to me. we're setting a single one. original choice was fine